### PR TITLE
Fix print to PDF and move formatting to stylesheet

### PR DESCRIPTION
**Pull request by captainsafia**
- Fix print to PDF feature
- Move styles to CSS

TBD:
- Fix bug where printToPDF only prints what is in viewport

### CODE REVIEW

**Comment by captainsafia** [comment]
Taking this out of draft mode to review these improvements. We still need to figure out why it is only printing one page.

**Comment by jarmokivekas** [comment]
I didn't yet have time to test this, but I'm happy to see activity on this issue. I kinda miss this feature, even keep a v0.23 installed just to have access to it :grin:

**Review by willingc — APPROVED** [review]
I tried this locally and it does capture part of the notebook as PDF and exports. I noticed that if I scrolled to the bottom of the notebook that it would export that section of the notebook.

Merging as is. We can work on the multipage PDF in a separate PR. Thanks @captainsafia :sunny:

File: applications/desktop/src/common/commands/by-menu/file.tsx [hunk 1]
```diff
@@ -168,40 +168,25 @@ export const ExportPDF: DesktopCommand<ReqContent> = {
...
-      data = await promisify(remote.getCurrentWindow().webContents.printToPDF)(
-        { printBackground: true },
-      );
-    }
-    finally {
-      // Restore the modified cells to their unexpanded state.
-      yield* unexpandedCells.map(
-        id => actions.toggleOutputExpansion({ id, ...props }),
-      );
+      const data = await remote.getCurrentWindow().webContents.printToPDF({ printBackground: false });
```
Comment (willingc) [inline applications/desktop/src/common/commands/by-menu/file.tsx:173]: If I had to guess I think this is the line that we'll need to update.

**Comment by captainsafia** [comment]
> I tried this locally and it does capture part of the notebook as PDF and exports. I noticed that if I scrolled to the bottom of the notebook that it would export that section of the notebook.

Yep! The issue is that it only prints what is currently on the page. I was able to workaround this by adding a `* { overflow: visible !important; }` style on the page for `@media print`. This prints both pages but also prints an additional set of 600 blank pages.

So it appears that there might be a sweet spot we need to land with the CSS here. It might be that we need to disable hiding the overflow on the cell components or something.

**Comment by willingc** [comment]
Gee thanks @captainsafia. I just about spit my drink out when you mentioned 600 blank pages! Thanks for the humor.

**Pull request merged**